ush/ioda/bufr2ioda/bufr2ioda_acft_profiles_prepbufr.py

```diff
@@ -28,6 +29,64 @@ def Compute_dateTime(cycleTimeSinceEpoch, dhr):
     return dateTime
 
 
+def Compute_typ_other(typ_other, var):
+
+    typ_var = copy.deepcopy(typ_other)
```

Comment:
question, is this always going to be "replace first digit by 1"? If so, can we simplify it to something like (pseudo code):

`typ_var[typ_var > 300 & typ_var < 400] = typ_var-200`? Or is that too difficult to code up?

Reply:
Hmmmm good point. I'll test it out!

Reply:
You were right. Made both programs a lot smaller. Updated and re-submitted.

Reply:
Thanks